x-pack/platform/plugins/shared/alerting/common/routes/r_rule/request/schemas/v1.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { rRuleRequestSchema } from './v1';
+
+describe('rRuleRequestSchema', () => {
```

Comment:
Could you please add validation for the other fields? `tzid`, `interval`, `until` and `count`

Reply:
Done!